packages/boxel/addon/components/boxel/progress-circle/index.gts

```diff
@@ -12,7 +12,7 @@ interface Signature {
   Element: HTMLDivElement;
   Args: {
     percentComplete: number;
-    size: number;
+    size?: number;
```

Comment:
If size is optional we should probably update the ProgressCircle usage and remove the `@required={{true}}` from size

Reply:
Updating...